devOpcuaSup/UaSdk/DataElementUaSdk.h

```diff
@@ -1052,7 +1063,7 @@ class DataElementUaSdk : public DataElement
 
     // Write array value for EPICS String / OpcUa_String
     long
-    writeArray (const char *value, const epicsUInt32 len,
+    writeArray (const char *value, epicsUInt32 len,
```

Comment:
Why not const anymore?

Reply:
There is no point declaring a non-pointer/non-reference primitive type function argument const. It is passed as a copy anyway. (The same for returning const primitive type results, btw.)
But if it is const, I cannot modify it in  https://github.com/dirk-zimoch/opcua/blob/4e909549527ff6017856a8bd27753f71221f5095/devOpcuaSup/UaSdk/DataElementUaSdk.cpp#L1339 and would need to make a local copy.

Reply:
Where static code analysis will make a big fuzz about changing a parameter and ask us to use a local variable...

Reply:
But: I see your point, I know.